### DISCUSSION TITLE
How to plot the Plankian Locus alongside SDs on CIE chromaticity diagram plots?

### DISCUSSION THREAD

**Discussion by ThomasWilshaw** [colour-science/colour#1287] [Q&A] [answered]
Hi,

Apolgies if I am missing something obvious but when plotting SDs on the chromaticity chart (e.g. `colour.plotting.plot_sds_in_chromaticity_diagram_CIE1931()`) is there a way to also show the Plankian locus on the same chart? Likewise is there a simple way to mix and match SDs, XYZ coordinates, primaries etc. all on one chart?

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @ThomasWilshaw,

Two ways:


```python
import colour

colour.plotting.plot_sds_in_chromaticity_diagram_CIE1931(
    [colour.SDS_ILLUMINANTS["D65"]],
    chromaticity_diagram_callable_CIE1931=lambda **kwargs: colour.plotting.plot_planckian_locus_in_chromaticity_diagram_CIE1931(
        [], **kwargs
    ),
)
```

```python
figure, axes = colour.plotting.plot_sds_in_chromaticity_diagram_CIE1931(
    [colour.SDS_ILLUMINANTS["D65"]],
    show=False,
)
colour.plotting.temperature.plot_planckian_locus(axes=axes)
```

Cheers,

Thomas

**Reply by ThomasWilshaw** [reply]
Thank you that worked perfectly

**Reply by ThomasWilshaw** [reply]
Out of curiosity why are the colour temperatures on the planckian locus plotted as lines rather than points?

**Reply by thomasmansencal** [reply]
Those are Iso Temperature lines where points on the line have the same colour temperature but not the same $D_{uv}$. We then talk about correlated colour temperature.

**Reply by ThomasWilshaw** [reply]
Thank you :)
